packages/ui/src/helpers/storage/fileUploader/utils/uploader.ts

```diff
@@ -27,13 +27,15 @@ export function uploadFile({
       progressCallback,
       errorCallback,
       completeCallback,
+      contentType: file.type || 'binary/octet-stream',
```

Comment:
We want all falsy values, not just null or undefined, to default to `binary/octet-stream`. Since falsy types are not valid MIME types.

This can be overridden by passing a `contentType` prop to the FileUploader if needed.